components/expenses/ExpenseForm.js

```diff
@@ -149,7 +152,14 @@ const validate = expense => {
       : requireFields(expense, ['description', 'payee', 'payee.name', 'payee.email']);
   }
 
-  const errors = isCardCharge ? {} : requireFields(expense, ['description', 'payee', 'payoutMethod', 'currency']);
+  let errors;
+  if (expense.payee.type === CollectiveType.COLLECTIVE) {
```

Comment:
Why do we have only `COLLECTIVE` here?

More generally, what is actually the goal of this change?

Reply:
Think I made a mistake here. This change is not needed. I've reverted this one. 🤔